src/generator/www/js/main.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var fd = new FormData(form);
+
+  var files ={
+    'sessionfile':  $( '#sessionfile' )[0].files[0],
+    'speakerfile':$( '#speakerfile' )[0].files[0],
+    'trackfile':$( '#trackfile' )[0].files[0] ,
+    'locationfile':$( '#locationfile' )[0].files[0]
+  }
+  
+  for (var key in files) {
+    //console.log(key, files[key]);
+    fd.append(key, files[key]);
+  }
+  var obj={
+    name:$name.val(),
+    email:$email.val(),
+    url :$url.val(),
+    files:fd
+    
+  }
+
+  $.ajax({
+    type: 'POST',
+    url:  ' ',
+    data: obj,
+    processData: false,
+    contentType: false,
+    
+    success :function(data){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['data' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l38)